static/js/lib/queries/digitalCredentials.js

```diff
@@ -0,0 +1,25 @@
+import { getCookie } from "../api"
+import type { DigitalCredential } from "../../flow/courseTypes"
```

Comment:
I think this type is missing?

Reply:
I used that earlier but later I just redirected from the response of the API, I just cleaned/removed it in the feedback commit since it wasn't needed now anyway.